### PR TITLE
some quick changes

### PR DESCRIPTION
**Pull request by redjoey**
I added to the help command - I put in help for loot

### CODE REVIEW

File: invadedhouse/commands.py [hunk 1]
```diff
@@ -62,14 +62,15 @@ def interpret_command(self, cmd: str):
             print('look - Open your eyes and remember what\'s around.')
             print('look at - Gaze upon anything nearby.')
             print('attack - enter a battle with a nearby monster.')
+            print('loot - Grab all the stuff from a chest!')
```
Comment (jsoconnell) [inline invadedhouse/commands.py:65]: Sweet, glad you remembered to add this to the help command!

File: invadedhouse/commands.py [hunk 2]
```diff
@@ -62,14 +62,15 @@ def interpret_command(self, cmd: str):
...
 
             print('Secret commands not included here!')
         elif cmd.startswith('pickup '):
             item_str = cmd.replace('pickup ','')
             if self.player.get_location().can_player_pickup(item_str):
                 item = self.player.get_location().remove_item(item_str)
                 self.player.pickup(item)
-                print(f'Picked up {item}!')
+                print(f'Picked up {item}!')
```
Comment (jsoconnell) [inline invadedhouse/commands.py]: Get rid of extra whitespace here

File: invadedhouse/commands.py [hunk 3]
```diff
@@ -125,6 +126,20 @@ def interpret_command(self, cmd: str):
             print('OMG YOU FOUND MY BRILLIANTLY WONDERFUL SECRET!!')
             warp_room = self.house.find_room_by_name('warp room')
             self.player.set_location(warp_room)
+        elif cmd.startswith('loot '):
+            item_str = cmd.replace('loot ','')
+            if self.player.get_location().can_player_pickup(item_str):
+                loot_items = self.player.get_location().get_item(item_str).loot()
+                for item in loot_items:
+                    self.player.pickup(item)
+                print(f'You looted items from {item_str}!')
```
Comment (jsoconnell) [inline invadedhouse/commands.py]: You should add the check if the chest is empty and say something different in that case.

File: invadedhouse/rooms.py [hunk 4]
```diff
@@ -352,7 +360,7 @@ def build_house(self):
 
         # set up all the exits and items and monsters for each room
         bedroom.set_exits([entry])
-        bedroom.set_items([sword, key, apple])
+        bedroom.set_items([trash_can])
```
Comment (jsoconnell) [inline invadedhouse/rooms.py:405]: Nice, good catch. (Not adding the items twice)

**Review by jsoconnell — APPROVED** [review]
Overall not bad, you can merge if you want but probably should make the change to say something different if chests are empty.

File: invadedhouse/commands.py [hunk 5]
```diff
@@ -125,6 +126,20 @@ def interpret_command(self, cmd: str):
...
+            else:
+                print('Can\'t loot. Try to loot a different loot.')
```
Comment (redjoey) [inline invadedhouse/commands.py]: loot command

File: invadedhouse/commands.py (hunk 2 above)
Comment (redjoey) [inline invadedhouse/commands.py]: don't know what I did here... Hmm?

Comment (jsoconnell) [reply invadedhouse/commands.py]: You added a space at the end.

**Pull request merged**